### PR TITLE
RDM-9280 Format Test JSON Data Files - Definition Store

### PR DESCRIPTION
**Pull request by skandala-cgi**
### JIRA link (if applicable) ###

https://tools.hmcts.net/jira/browse/RDM-9280

### Change description ###

Format Test JSON Data Files - Definition Store

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: aat/src/aat/resources/features/F-083/S-359.td.json [hunk 1]
```diff
@@ -1,119 +1,121 @@
...
-					"label": "Complex DateTime",
-					"order": 10,
-					"role": null,
-					"display_context_parameter": null,
-					"show_condition": null
-				}
-			]
-		}
-	}
+  "_guid_": "S-359",
```
Comment (MSancaktutar) [inline aat/src/aat/resources/features/F-083/S-359.td.json:2]: auto format with tabs not applied correctly to this and some other files.

**Review by MSancaktutar — CHANGES_REQUESTED** [review]
Nice good work, but auto format with tabs not applied correctly to some files.
also, pls see my refinement commits for later similar tasks. Many thanks.

File: aat/src/aat/resources/features/F-083/S-359.td.json (hunk 1 above)
Comment (skandala-cgi) [reply aat/src/aat/resources/features/F-083/S-359.td.json]: updated

**Review by MSancaktutar — APPROVED** [review]
LGTM

**Pull request merged**